### ISSUE TITLE
List of coverage providers

### ISSUE THREAD

**Issue by blueyed** [google/vim-coverage#12]
Are there any coverage providers already?
Would be nice to have list somewhere (README or wiki)?!

**Comment by dbarnett** [comment]
None yet. =(
This plugin doesn't serve much purpose until #6, #7, #8, or some other coverage provider gets implemented. After that, yeah, we should definitely list available providers in built-in help, README, and/or wiki.

@stgpetrovic We might want to add some kind of note to the README / help about the current lack of coverage providers for public consumption, so people who aren't planning on hacking on it don't waste their time trying to use it in the mean time.

**Comment by mattsoftware** [comment]
I agree with @dbarnett . I just installed this thinking I could get it to work with jest results, however its not until AFTER the plugin was installed etc that I found out the only provider is coverage.py

README.md is the perfect place to put the current list of providers, either included and/or links to other providers that can be used with this plugin.

**Comment by stgpetrovic** [comment]
I agree with this. I'll add it as soon as I reach the land (I'm on vacation). I can also add a plugin or two for these PRs.
